src/index.js

```diff
@@ -15,7 +15,7 @@ function single(key, dep) {
 }
 
 function multi(keys, dep, base) {
-	base = base || dep.split('/').pop().replace(/\W/g, '_') + '$1'; // uniqueness
+	base = base || dep.split('/').pop().replace(/\W/g, '_') + '$' + NAMED.num++; // uniqueness
```

Comment:
Wouldn't it more sense to use an actual variable instead of monkey-patching a RegExp?

Reply:
I had it originally, but doesn't really matter. It's a a custom key & has to be reset every invocation of the function anyway